testsuite/MDAnalysisTests/analysis/test_pca.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python2.7
+
+import MDAnalysis
+import MDAnalysis.analysis.pca as pca
```

Comment:
This call here is tripping up the minimal build (to even import pca you need scipy).

Ways around this I can see are:
1)
try/except block around the import, so 
``` python
try:
    import MDAnalysis.analysis.pca as pca
except ImportError:
    pass
```
Then because the setUp is decorated, the `test_*` methods never get called anyway, so the `pca.` import isn't ever required.

2)
If that doesn't work, put the pca import inside the class, so
``` python
def setUp(self):
    import MDAnalysis.analysis.pca as pca
    self.PCA = pca.PCA

def test_this(self):
    p = self.PCA(stuff)
```